src/test/java/com/prgrms/commandLineApplication/customer/CustomerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.prgrms.commandLineApplication.customer;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class CustomerTest {
+
+  @ParameterizedTest
+  @CsvSource(value = {"qkrdmswl1018@naver.com|박은지", "test@gmail.com|사용자2", "email@naver.com|사용자2"}, delimiter = '|')
+  @DisplayName("올바른 형식의 이메일과 이름을 작성했을 경우 customer 생성 성공")
```

Comment:
```suggestion
  @DisplayName("올바른 형식의 이메일과 이름을 작성했을 경우 고객이 생성된다")
```

테스트 메서드나 DisplayName은 `가정-행위-결과`의 완전한 문장으로 작성해주는 편이 가독성이 좋아요

Reply:
07a4baf4442776cfe9a50694b9fec38966e41fe3